test/functional/wallet_fundrawtransaction.py

```diff
@@ -1312,6 +1313,38 @@ def test_weight_calculation(self):
 
         self.nodes[2].unloadwallet("test_weight_calculation")
 
+    def test_weight_limits(self):
+        self.log.info("Test weight limits")
+
+        self.nodes[2].createwallet("test_weight_limits")
+        wallet = self.nodes[2].get_wallet_rpc("test_weight_limits")
+
+        outputs = []
+        for _ in range(1472):
+            outputs.append({wallet.getnewaddress(address_type="legacy"): 0.1})
+        txid = self.nodes[0].send(outputs=outputs)["txid"]
+        self.generate(self.nodes[0], 1)
+
+        # 272 WU per input (273 when high-s); picking 1471 inputs will exceed the max standard tx weight.
```

Comment:
From `(273 when high-s)`, I assume this considers a signed input but the docs of `createrawtransaction` and `fundrawtransaction` say that these commands don't generate a signed transaction. So this max weight check on picking 1472 inputs of 273 WU each internally seems to be dependent only on the `weight` parameter passed to `fundrawtransaction` RPC, which is dependent on the user input. 
Is there not any other internal worst-case scenario treatment of the inputs weights that might not be dependent on user input?

Reply:
yeah, good eye. `send()` and `walletprocesspsbt()` will also fail. Will add coverage for one of them.